values.yaml

```diff
@@ -66,6 +66,15 @@ enableEmbeddedQueue: false
 ## @param appContextRoot Release context root.
 ##
 appContextRoot: /
+## @param logbackGlobalLoggingLevel Global logging level. Possible values: "trace", "debug", "info", "warn", "error"
+##
+logbackGlobalLoggingLevel: "info"
+## @param logbackScanEnabled Enables scanning of logback.xml.
+##
+logbackScanEnabled: true
+## @param logbackScanPeriod Interval for checking logback.xml configuration.
+##
+logbackScanPeriod: "30 seconds"
```

Comment:
Maybe it would be better to split names and have all props under logback section, for example:

logback:
  globalLoggingLevel:
  scanEnabled:
  scanPeriod:

Reply:
Done :)